service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/mapper/EncounterComponentsMapperTest.java

```diff
@@ -78,6 +78,10 @@ public class EncounterComponentsMapperTest {
     private static final String INPUT_BUNDLE_TOPIC_NO_CATEGORIES = TEST_DIRECTORY + "input-bundle-17-topic-no-categories.json";
     private static final String EXPECTED_COMPONENTS_TOPIC_NO_CATEGORIES = TEST_DIRECTORY
         + "expected-components-17-topic-no-categories.xml";
+    public static final String INPUT_BUNDLE_TOPIC_RELATED_PROBLEM_INVALID_EXTENSION_URL =
+        TEST_DIRECTORY + "input-bundle-18-related-problem-invalid-problem-extension.json";
+    public static final String EXPECTED_COMPONENTS_TOPIC_RELATED_PROBLEM_INVALID_EXTENSIONS =
+        TEST_DIRECTORY + "expected-components-18-related-problem-invalid-extension.xml";
```

Comment:
Do these need to be static constants if they are only being used in one test? 😄

Reply:
not really